app/views/notifier/password_reset_instructions.erb

```diff
@@ -1,8 +1,6 @@
 A request to reset your password has been made.
 If you did not make this request, simply ignore this email.
-If you did make this request just click the link below:
-
-<%= edit_password_reset_url(@token) %>
+If you did make this request <%= link_to "click here to reset it", edit_password_reset_url(@token) %>.
 
 If the above URL does not work try copying and pasting it into your browser.
 If you continue to have problem please feel free to contact us.
```

Comment:
Unrelated typo since I happen to be reading: should be "If you continue to have problems"